File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        d={}
        for i in range(len(nums)):
            a=target-nums[i]
            if a in d:
                return [d.get(a),i]
            d[nums[i]]=i
        return []

File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let obj={};
    for(let i=0;i<nums.length;i++)
    {
        let a=target-nums[i];
        if(a in obj)
        {
            return [obj[a],i];
        }
        obj[nums[i]]=i;
    }

};